pkg/api/component_report.go

```diff
@@ -1022,9 +1027,13 @@ func getNewCellStatus(testID apitype.ComponentReportTestIdentification,
 		newCellStatus.status = reportStatus
 	}
 	if reportStatus < apitype.MissingSample {
+		logger.Infof("adding regressed test")
```

Comment:
This shows the 8 calls happening at various times:

❯ cat allout.txt| grep openshift-tests:fefc532f9a1e9928f7b22d188a7d4799  | grep "adding regressed" | grep Build
time="2024-04-12T13:52:44.351-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:44.373-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:44.601-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:44.806-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:44.815-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:45.139-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:45.177-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"
time="2024-04-12T13:52:45.249-03:00" level=info msg="adding regressed test" columnID="{ovn  amd64 vsphere }" row="{Build    }" testID="openshift-tests:fefc532f9a1e9928f7b22d188a7d4799"